autosubmitconfigparser/config/configcommon.py

```diff
@@ -1185,13 +1210,73 @@ def check_conf_files(self, running_time=False, force_load=True, no_log=False):
         except AutosubmitCritical as e:
             # In case that there are critical errors in the configuration, Autosubmit won't continue.
             if running_time is True:
-                raise AutosubmitCritical(e.message, e.code, e.trace)
+                raise
             else:
                 if not no_log:
                     Log.warning(e.message)
-        except Exception as e:
-            raise AutosubmitCritical(
-                "There was an error while showing the config log messages", 7014, str(e))
+        except Exception:
+            raise
+
+    def validate_wallclock(self) -> str:
+        """
+        Validate the wallclock time for each job against the platform's maximum wallclock time.
+
+        :return: Error message if any job exceeds the platform's wallclock time, otherwise an empty string.
+        :rtype: str
+        """
+
+        def _calculate_wallclock(wallclock: str) -> int:
```

Comment:
I think `timedelta(...).total_seconds()` will return a `float`, not `int`, 

![image](https://github.com/user-attachments/assets/6e7927fe-f244-477e-848b-72a54babef7b)